src/kernel/src/fs/dev/sblsrv.rs

```diff
@@ -0,0 +1,41 @@
+use crate::fs::{make_dev, Mode, VPath};
+use crate::ucred::{Gid, Uid};
+use macros::vpath;
+use std::fmt::{Display, Formatter};
+use std::sync::Arc;
+
+use super::{CdevSw, DriverFlags, MakeDev};
+
+/// An implementation of `/dev/sbl_srv`.
+#[derive(Debug)]
+pub struct SblService {}
+
+impl SblService {
+    pub const PATH: &'static VPath = vpath!("/dev/sbl_srv");
+
+    pub fn new() -> Arc<Self> {
+        let sbl_srv_cdevsw = Arc::new(CdevSw::new(
+            DriverFlags::from_bits_retain(0x80000000),
```

Comment:
This is DriverFlags::NEEDS_MINOR

Reply:
NEEDS_MINOR is 0x00800000, according to FreeBSD that is D_INIT